src/main/java/Main.java

```diff
@@ -6,7 +6,31 @@
 
     public static void main(String[] args) {
 
-        System.out.println("Hello World! ");
+        int tab2[][]={{21,50,50},{22,90,90},{23,130,130},{24,220,260},{25,300,400},{26,400,600},{27,430,630},{28,460,660},
+                {29,490,690},{30,520,720},{31,700,720},{32,900,1350},{33,990,1440},{34,1250,1800},{36,1500,2200}};
+        int wartoscReki,etap;
+
+        System.out.println("Wartosc punktowa reki: ");
+        Scanner input=new Scanner(System.in);
+        wartoscReki=input.nextInt();
+
+        while(wartoscReki<21)
+        {
+            System.out.println("Wartosc nie moze byc mniejsza niz 21!");
+            wartoscReki=input.nextInt();
+        }
+
+        System.out.println("Przed partią - 1, Po partii - 2");
```

Comment:
Wydaje mi się, że lepiej by było jakby użytkownik wpisywał słownie "przed" albo "po", a nie 1 lub 2. Wtedy przyda się zrobić switch-case'a, żeby zobaczyć czy użytkownik wpisał dobre słowo.

Reply:
Wymyśliłam opcje 1 i 2, aby potem, w metodzie wyliczającej z tablicy szukaną sumę, mieć zmienną odpowiadającą kolumnie nr 1 lub nr 2. Powiedz mi Adam, jak Ty widzisz użycie instrukcji switch-case,  wiem ze w Default wpisałabym komunikat i ponowne pobranie stringa, a w case "przed" i case "po" (albo dodatkowo te wyrazy pisane z duzej litery i duzymi literami), jakie instrukcje w tych case-ach?  żadnej instrukcji nie wpisać? Uznać, ze w takiej sytuacji po prostu jest dobrze pisane? :)

Reply:
Ja tylko skomentuje: 
Jest sporo sposobów aby nie sprawdzac pisowni. Np. https://www.tutorialspoint.com/java/java_string_touppercase.htm
Case tez odpowiada pewnym wartoscia intowym wiec z logiki programu moze byc dalej 1 i 2, po prostu interfejs uzytkownika jest czytelniejszy. A najprostsza mozliwa instukcja to ustawic zmienna na odpowiedni parametr.